src/momento/simple_cache_client.py

```diff
@@ -22,13 +23,27 @@ def __init__(
         default_ttl_seconds: int,
         data_client_operation_timeout_ms: Optional[int],
     ):
+        try:
+            # If the synchronous client is used inside an async application,
+            # use the event loop it's running within.
+            loop: asyncio.AbstractEventLoop = asyncio.get_running_loop()
```

Comment:
yeah this would be a mistake, but the impact of using the wrong loop in this client is that the coroutines throw when resumed on the wrong loop, or if you always re-wrap in a different loop it would ¡¡¡block the rest of the user's coroutines¡¡¡ both of which seem really awful. So we try to use the one the user intends to use if they're already using one and if they're not we go ahead and make one. There might be a reason we should use get_event_loop() instead but this one's maybe more idiomatic and they're both kind of hacky workarounds to enable graceful degradation in an awkward downstream app config.